composer.json

```diff
@@ -18,7 +18,8 @@
 
         "smarty/smarty": "^3.1",
         "geshi/geshi": "^1.0.9",
-        "setasign/fpdf": "^1.8"
+        "setasign/fpdf": "^1.8",
+        "planetteamspeak/ts3-php-framework": "^1.1"
```

Comment:
I added this as an optional suggestion and not as requirement, as it is only required by this single module.
Similar to how the SNMP-module was handled for the NOC